### DISCUSSION TITLE
Classic projects allow Notes with body text in the board, while Draft card don't have this option

### DISCUSSION THREAD

**Discussion by srbarrios** [community/community#54250] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

Hi all,

After migrating my Classic Project board to the new style, all the notes were converted to drafts, but they only show the title in the board. It doesn't expand to show all the body of the draft, as it was possible previously.

That was quite handy for some notes, like the planned absences in the team.
Can we please consider porting that feature? 

**Example**:

Classic Project:
<img width="336" alt="Captura de pantalla 2023-05-02 a las 14 50 26" src="https://user-images.githubusercontent.com/2827771/235671090-0fc12782-899e-4eb4-bfbd-19cc71864cd2.png">

Migrated project:
<img width="347" alt="Captura de pantalla 2023-05-02 a las 14 48 59" src="https://user-images.githubusercontent.com/2827771/235670773-10fcb7d0-7476-4bd9-8d69-45f8e2c1421c.png">

Thank you

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by OleMchls** [comment]
I'm currently doing the same exercise, some info we had directly **on** our board before I migrated to the _Readme_, but that's not as visible as having the info right on the board.

To me, a Draft is something different than a Note and I miss the classic Notes deeply. We've used them just like sticky notes on a whiteboard and with the new project that use-case is gone. I'd love to have them back, I hope it's alright to just piggyback on this issue.
